packages/builder/src/steps/import.ts

```diff
@@ -47,8 +50,19 @@ export default {
   configInject(ctx: ChainBuilderContextWithHelpers, config: Config) {
     config = _.cloneDeep(config);
 
-    config.source = _.template(config.source)(ctx);
-    config.preset = _.template(config.preset)(ctx) || 'main';
+    
+    const packageRef = new PackageReference(_.template(config.source)(ctx));
+
+    // If both definitions of a preset exist, its a user error.
+    if (config.preset && packageRef.preset) {
```

Comment:
Need to rewrite this, this check is redundant since packageRef.preset always defaults to main.